### PR TITLE
Migrated more tests

### PR DESCRIPTION
**Pull request by narmaku**

### CODE REVIEW

File: test_suite/cloud/test_aws.py [hunk 1]
```diff
@@ -397,6 +397,54 @@ def test_yum_plugins(self, host):
...
+                f'Expected configuration was not found in "{file_to_check}"'
+
+    @pytest.mark.run_on(['rhel8.5', 'rhel8.6', 'rhel9.0'])
+    def test_dracut_conf_xen(self, host):
+        """
+        BugZilla 1849082
+        Add ' xen-netfront xen-blkfront ' to '/etc/dracut.conf.d/xen.conf' in x86 AMIs prior RHEL-8.4.
+        Using image builder from RHEL-8.5, add ' nvme xen-blkfront ' to '/usr/lib/dracut/dracut.conf.d/ec2.conf'.
+        This is not required in arm AMIs.
+        """
```
Comment (jabia99) [inline test_suite/cloud/test_aws.py:423]: I would not remove the ticket reference `#COMPOSER-1096` from the test. I think every context we can provide has value. Maybe we can include it on the docstring

**Review by jabia99 — APPROVED** [review]
LGTM!

File: test_suite/cloud/test_aws.py (hunk 1 above)
Comment (narmaku) [reply test_suite/cloud/test_aws.py:426]: Good point! I forgot to add it here.

**Review by jabia99 — APPROVED** [review]
Looks great!

The subscription manager test is huge! Did you test it on all sample images?

**Comment by narmaku** [comment]
> Looks great!
> 
> The subscription manager test is huge! Did you test it on all sample images?

Thanks! Yes I did, but the tests will fail in all AMIs except some of them that are ready to be published. E.g. the one that Frank shared with us via chat. In that AMI, there's a timeout while checking the subscription status, which is always unknown. I think it's something related to the environment and not the test itself. So we will follow-up on this with Frank directly.

**Pull request merged**